mailpoet/assets/js/src/segments/dynamic/dynamic-segments-filters/fields/woocommerce/purchased-tag.tsx

```diff
@@ -0,0 +1,92 @@
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from 'react';
+import { filter } from 'lodash/fp';
+import { Select } from 'common/form/select/select';
+import { MailPoet } from 'mailpoet';
+import { ReactSelect } from 'common/form/react-select/react-select';
+import { __ } from '@wordpress/i18n';
+import { storeName } from '../../../store';
+import {
+  AnyValueTypes,
+  FilterProps,
+  SelectOption,
+  WindowProductCategories,
+  WooCommerceFormItem,
+} from '../../../types';
+
+export function validatePurchasedTag(formItems: WooCommerceFormItem): boolean {
+  const purchasedCategoryIsInvalid =
```

Comment:
Perhaps the name of variable here could be `purchasedTagIsInvalid` or maybe `filterConfigIsInvalid`. 
The current name refers to different filter type so it might be a little confusing.

Reply:
Thank you for catching this! I really thought I'd caught all references to category but this slipped through. Updated in b2adace9524cefc5f4d10b6f091cd9fae342af56.